source/utils/Mutable.ts

```diff
@@ -0,0 +1,3 @@
+export type Mutable<T> = {
```

Comment:
The file should be named `types.ts`. I don't want a separate file for each simple type utility.

Reply:
yeah, that's working for me

Done ✅